main.tf

```diff
@@ -56,7 +60,7 @@ module "db_instance" {
   snapshot_identifier = "${var.snapshot_identifier}"
 
   vpc_security_group_ids = ["${var.vpc_security_group_ids}"]
-  db_subnet_group_name   = "${module.db_subnet_group.this_db_subnet_group_id}"
+  db_subnet_group_name   = "${local.db_subnet_group_name}"
```

Comment:
When `var.db_subnet_group_name` is not empty it will still respect `var.create_db_subnet_group` and probably create unused db_subnet_group.

Can you change argument `count` in `db_subnet_group` ([line 11](https://github.com/terraform-aws-modules/terraform-aws-rds/pull/38/files#diff-7a370d8342e7203b805911c92454f0f4R11))  to respect `var.create_db_parameter_group` only if `var.db_subnet_group_name` is not set?

Reply:
👍

Reply:
Initially I put the condition on `count` value, but it felt a bit complicated to I moved it to locals.

Let me know if you want to move it back to `count`.